src/content/docs/release-notes/agent-release-notes/python-release-notes/python-agent-90101.mdx

```diff
@@ -0,0 +1,31 @@
+---
+subject: Python agent
+releaseDate: '2023-10-19'
+version: 9.1.1
+downloadLink: 'https://pypi.python.org/pypi/newrelic'
+features: ['Add get_package_version caching']
+bugs: ['Fix redis generator instrumentation']
+security: []
+---
+
+## Notes
+
+This release of the Python agent fixes generator instrumentation issue with Redis and adds caching to package versions to address some performance issues.
+
+Install the agent using `easy_install/pip/distribute` via the [Python Package Index](https://pypi.python.org/pypi/newrelic) or download it directly from the [New Relic download site](https://download.newrelic.com/python_agent/release).
+
+## New features
+
+* **Add caching to `get_package_version` to increase performance**
+The agent removes redundant calls to find the package version which will address performance issues.
+
+## Bug fixes
+
+* **Fix generator and async generator instrumentation issue in Redis**
+Fixed instrumentation of generators in `Redis.asyncio`.
```

Comment:
@lrafeei Can we combine these into just one sentence?

Reply:
Sure--I don't have strong feelings on the matter